config_samples/lido_dao_holesky_config.json

```diff
@@ -111,5 +112,38 @@
             "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
             "relative_root": "contracts"
         }
+    },
+    "binary_checking": {
+	"use": 1,
```

Comment:
```suggestion
	"enable": true,
```

Reply:
Should be handled in the code accordingly.

Reply:
done